docs/references.md

```diff
@@ -38,3 +38,13 @@
 
 - [Ralf Gommers - Python packaging in 2021 - pain points and bright spots (2021)](https://labs.quansight.org/blog/2021/01/python-packaging-brainstorm/)
 
+
+## General packaging core concepts
```

Comment:
proposed tweak, one of:
- General Python packaging core concepts and howto's
- General Python packaging content